docs/aws-professional/services.md

```diff
@@ -52,7 +52,8 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
 | Virtual Server disk infrastructure | Elastic Block Store (EBS)                                 | [Azure Storage Disk—Page Blobs (for VHDs or other random-write type data)](/rest/api/storageservices/fileservices/understanding-block-blobs--append-blobs--and-page-blobs#about-page-blobs) <br/><br/>[Azure Storage Disks—Premium Storage](https://azure.microsoft.com/services/storage/disks/) | SSD storage optimized for I/O intensive read/write operations. For use as high performance Azure virtual machine storage.                                                                                                                                                                           |
 | Shared file storage                | Elastic File System                                       | [Azure Files  (file share between VMs)](https://azure.microsoft.com/services/storage/files/)                                                                                                                                                                                                                     | Provides a simple interface to create and configure file systems quickly, and share common files. It’s shared file storage without the need for a supporting virtual machine, and can be used with traditional protocols that access files over a network.                                          |
-| Archiving—cool storage             | S3 IA Glacier                                             | [Azure Storage—Standard Cool](/azure/storage/storage-blob-storage-tiers)                                                                                                                                                                                                                     | Cool storage is a lower cost tier for storing data that is infrequently accessed and long-lived.                                                                                                                                                                                                    |
+| Archiving—cool storage             | S3 Infrequent Access (IA)                               | [Azure Storage—Standard Cool](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers)                                                                                                                                                                                                                     | Cool storage is a lower cost tier for storing data that is infrequently accessed and long-lived.                                                                                                                                                                                                    |
```

Comment:
You should always remove the `en-us` piece of the URL so that it resolves automatically to the user's browser locale.

Reply:
Yep. Also in the Docs platform, we can format links as `/azure/....` and the publishing system resolves them. I'll fix them for the current PR.